Vox nerfs the big umpf

Rayfod opened the pull request:
Vox nerfs: Kosmag removed, Armalis changed to one, and the sniper rifle nerfed so that it **cannot** stun anymore while giving it back its original ammo.
Nerfed the dark matter gun, to charge a focused shot, every 12 seconds.
These changes were talked about with AntonKR before they were made.
Final confirmation from Carl would be good.
Let's see how this goes.
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Rayfod committed:
Vox nerfs the big umpf

Vox nerf, Kosmag removed, Armalis changed to one, and the sniper rifle further nerfed

Anton-Kr commented:
Ill ask for Carl’s approval prior to merging this.

Rayfod committed:
More recharge time

BurpleBineapple commented on maps/away/voxship/voxship_jobs.dm:
```diff
@@ -210,8 +210,8 @@
 /obj/item/weapon/gun/energy/sniperrifle/vox
```
Hot take, but I'd say remove this entirely. A self-recharging practically infinite ammo 1 shot knockout gun is pretty meme.

Rayfod replied:
I changed the stun instead of removing the gun.

Rayfod committed:
I did a stupid

Rayfod committed:
Sniper does not stun, 12 second rechare time for the focused shot

This makes the voxes able to "have" the sniper rifle, without the stun gameplay

Rayfod committed:
Merge branch 'Vox-nerfs-102' of https://github.com/Rayfod/BoH-Bay into Vox-nerfs-102

Rayfod committed:
Stupid update system

Rayfod committed:
Better desc

Rayfod committed:
Merge branch 'Vox-nerfs-102' of https://github.com/Rayfod/BoH-Bay into Vox-nerfs-102

Rayfod committed:
I hate github, I hate github

Rayfod committed:
The Vox-lore man asked if I could change the books

Rayfod committed:
Merge branch 'Vox-nerfs-102' of https://github.com/Rayfod/BoH-Bay into Vox-nerfs-102

Rayfod committed:
Update voxship_jobs.dm

Rayfod commented:
#1349 are going to take my nerfs into his PR.
